contrib/scriptlets/deb/postinst

```diff
@@ -26,7 +26,8 @@ helpMessage="If you need help using the app, use the command 'nordvpn --help'."
 
 # update configuration and shared library cache for linker to find .so files
 echo "/usr/lib/nordvpn" > /etc/ld.so.conf.d/nordvpn.conf
-ldconfig
+# NOTE: sudo is required here even though the script is run with EUID 0
+sudo ldconfig
```

Comment:
What if this is installed on sudoless environment?

Reply:
Hmm, good point. So it can be installed on sudoless env only when run with root. I can check if sudo is available and then use it, if it's not available, then it's run with root. I need to check if there are no issues there